plugin/trino-cassandra/pom.xml

```diff
@@ -157,13 +157,6 @@
             <groupId>io.trino</groupId>
             <artifactId>trino-testing</artifactId>
             <scope>test</scope>
-            <exclusions>
-                <exclusion>
-                    <!-- not needed for pinot, and results with conflict regarding netty and apache httpclient -->
-                    <groupId>io.trino</groupId>
-                    <artifactId>trino-exchange</artifactId>
```

Comment:
This belongs to "Remove trino-testing dependency on trino-exchange". Can you please squash?

Reply:
Done!